EPlast/EPlast.WebApi/Controllers/AuthController.cs

```diff
@@ -123,6 +131,33 @@ public async Task<IActionResult> GoogleLogin(string googleToken)
             return BadRequest();
             
 
+        }
+
+        [HttpPost("signin/facebook")]
+        [AllowAnonymous]
+        public async Task<IActionResult> FacebookLogin([FromBody] FacebookUserInfo userInfo)
+        {
+            try
+            {
+                var user = await _authService.FacebookLoginAsync(userInfo);
+
+                if (user == null)
+                {
+                    return BadRequest();
+                }
+
+                var generatedToken = await _jwtService.GenerateJWTTokenAsync(user);
+
+                return Ok(new { token = generatedToken });
+            }
+            catch (Exception exc)
+            {
+                _loggerService.LogError(exc.Message);
+            }
+
+            return BadRequest();
+
```

Comment:
```suggestion
                var user = await _authService.FacebookLoginAsync(userInfo);
                if (user == null)
                {
                    return BadRequest();
                }
                var generatedToken = await _jwtService.GenerateJWTTokenAsync(user);

                return Ok(new { token = generatedToken });

```
Don't we have default error handling implemented?